nengo/base.py

```diff
@@ -3,6 +3,7 @@
 
 import numpy as np
 
+import nengo
```

Comment:
I'm a bit surprised that this doesn't give a circular import. :astonished: 
But if it works, great!